mappings/net/minecraft/client/toast/SystemToast.mapping

```diff
@@ -16,24 +31,52 @@ CLASS net/minecraft/unmapped/C_vksdgrxp net/minecraft/client/toast/SystemToast
 		ARG 1 worldName
 	METHOD m_fnkorczd add (Lnet/minecraft/unmapped/C_eqfjupcc;Lnet/minecraft/unmapped/C_vksdgrxp$C_ozahoshp;Lnet/minecraft/unmapped/C_rdaqiwdt;Lnet/minecraft/unmapped/C_rdaqiwdt;)V
 		ARG 0 manager
+		ARG 1 id
 		ARG 2 title
 		ARG 3 description
 	METHOD m_jwgfdkzb addWorldDeleteFailureToast (Lnet/minecraft/unmapped/C_ayfeobid;Ljava/lang/String;)V
 		ARG 0 client
 		ARG 1 worldName
+	METHOD m_kewnuctq addChunkLoadFailureToast (Lnet/minecraft/unmapped/C_ayfeobid;Lnet/minecraft/unmapped/C_ynrszrtu;)V
+		ARG 0 client
+		ARG 1 chunkPos
+	METHOD m_ltbbvrya addLowDiscSpaceToast (Lnet/minecraft/unmapped/C_ayfeobid;)V
+		ARG 0 client
 	METHOD m_nvohfuwc setContent (Lnet/minecraft/unmapped/C_rdaqiwdt;Lnet/minecraft/unmapped/C_rdaqiwdt;)V
 		ARG 1 title
 		ARG 2 description
-	METHOD m_saimreyq addPackCopyFailure (Lnet/minecraft/unmapped/C_ayfeobid;Ljava/lang/String;)V
+	METHOD m_saimreyq addPackCopyFailureToast (Lnet/minecraft/unmapped/C_ayfeobid;Ljava/lang/String;)V
 		ARG 0 client
 		ARG 1 directory
 	METHOD m_vkmbhoha show (Lnet/minecraft/unmapped/C_eqfjupcc;Lnet/minecraft/unmapped/C_vksdgrxp$C_ozahoshp;Lnet/minecraft/unmapped/C_rdaqiwdt;Lnet/minecraft/unmapped/C_rdaqiwdt;)V
 		ARG 0 manager
+		ARG 1 id
 		ARG 2 title
 		ARG 3 description
+	METHOD m_vqhfyjbx addChunkSaveFailureToast (Lnet/minecraft/unmapped/C_ayfeobid;Lnet/minecraft/unmapped/C_ynrszrtu;)V
+		ARG 0 client
+		ARG 1 chunkPos
 	METHOD m_wvhbcnhf create (Lnet/minecraft/unmapped/C_ayfeobid;Lnet/minecraft/unmapped/C_vksdgrxp$C_ozahoshp;Lnet/minecraft/unmapped/C_rdaqiwdt;Lnet/minecraft/unmapped/C_rdaqiwdt;)Lnet/minecraft/unmapped/C_vksdgrxp;
 		ARG 0 client
+		ARG 1 id
 		ARG 2 title
 		ARG 3 description
+	METHOD m_yghdjgwj hide ()V
+	METHOD m_zfyvjxrz (Lnet/minecraft/unmapped/C_sedilmty;Lnet/minecraft/unmapped/C_eqfjupcc;J)Lnet/minecraft/unmapped/C_zqhkdxaf$C_sakiolwc;
+		ARG 1 graphics
 	METHOD m_zstagntz getTextAsList (Lnet/minecraft/unmapped/C_rdaqiwdt;)Lcom/google/common/collect/ImmutableList;
 		ARG 0 text
+	CLASS C_ozahoshp Id
```

Comment:
to expand my position a little, usually `id` is something more internal while `type` is something more user-facing

Reply:
The issue of Type is that it implies that it can be recycled between toasts of a certain type while in fact? It's more of an identifier for one specific toast; The SystemToast.Type name had made me mistakenly use a vanilla toast "type" for a modded toast when in fact it shouldn't have been used that way; This is why I kinda resists against `Type`;

I guess this refactor could be reverted though, it's just that yeah, the original name bothers me; it feels misleading

Reply:
oh girl i did the exact same thing as you in my mod is that not what i'm supposed to do

Reply:
> oh girl i did the exact same thing as you in my mod is that not what i'm supposed to do

You are not! You are supposed to create your own SystemToast.<Thing> and use it for toasts; If you use a Vanilla one? If the Vanilla one and your toast attempts to be added at the same time? One's gonna overwrite the other since they share the same id! Hence why am proposing the Mojmap-inspired `SystemToast.Id` (moj's way more verbose though): because it is effectively the id of a toast, and it is the method which allows toasts to be differentiated when added

Reply:
Could you add a commend to `Toast#getId` clarifying that toasts with the same id will override one another?

Reply:
Sure! I can Javadoc something; not sure if at `Toast#getId`, but I can try something

Reply:
I opted for a Javadoc on SystemToast.Id itself

Reply:
Hm, doesn't it apply to any object returned as an id from `Toast#getId`, including the default `ID` in `Toast`?

I think it would make more sense to put it there, with an `@see` pointing to `SystemToast.Id`.